apps/client/src/app/import/import.component.ts

```diff
@@ -232,6 +227,22 @@ export class ImportComponent implements OnInit {
     }
   }
 
+  public async getList(importFileModel) {
+
+    let url = `/api/fhir/${importFileModel.resource.resourceType}`;
+
+    if (importFileModel.resource.id) {
```

Comment:
Consider doing 
`if (!importFileModel.resource || !importFileModel.resource.resourceType || !importFileModel.resource.id) return;` at the top of the method.
I believe it is possible that if a file fails to parse into a resource, `importFileModel.resource` could be null. And either way, `resourceType` and `id` are both needed to call the API endpoint.